integreat_cms/api/v3/events.py

```diff
@@ -161,7 +161,7 @@ def transform_event_recurrences(event_translation, today):
             days=settings.API_EVENTS_MAX_TIME_SPAN_DAYS
         ):
             break
-        if recurrence_date < today or recurrence_date == start_date:
+        if recurrence_date < today:
```

Comment:
Adding the first occurrence to this function definitely makes sense! The problem is that it doesn't quite work when the recurring event goes on for several days and the start_date was yesterday or earlier.
But I think this should be a rather rare edge case in comparison to the bug that is fixed by this PR, so I guess we should merge and fix this problem later.